lib/api/CDataFrameTrainBoostedTreeRunner.cc

```diff
@@ -168,6 +168,8 @@ CDataFrameTrainBoostedTreeRunner::CDataFrameTrainBoostedTreeRunner(
 
```

Comment:
You are missing `theReader.addParameter(...)` above.

Reply:
Done in [e294c87](https://github.com/elastic/ml-cpp/pull/2153/commits/e294c872d42482e2b16ebee3e9a3866e002d72d1).